superset/assets/visualizations/big_number.js

```diff
@@ -39,7 +48,7 @@ function bigNumberVis(slice, payload) {
   const dateExt = d3.extent(data, (d) => d[0]);
   const valueExt = d3.extent(data, (d) => d[1]);
 
-  const margin = 20;
+  const margin = 30;
```

Comment:
Should the margin be relative? 30 becomes relatively big when using the smallest 1by1 cell in the dashboard